<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpdrGuiBasicInputForm</class>
 <widget class="QWidget" name="SpdrGuiBasicInputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <item row="0" column="0">
    <widget class="QLabel" name="labelInput">
     <property name="text">
      <string>Input directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="inputPlaceholder" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Type in the directory holding your up-to-date data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonInput">
     <property name="toolTip">
      <string>Browse to directory holding your up-to-date data</string>
     </property>
     <property name="text">
      <string>Select dir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelOutput">
     <property name="text">
      <string>Output directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="outputPlaceholder" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Specify the directory that should be brought up to date with the input directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonOutput">
     <property name="toolTip">
      <string>Browse to directory that should be brought up to date with the input directory</string>
     </property>
     <property name="text">
      <string>Select dir</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Basic options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelLogLevel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Log level</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLogLevel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Use this control to specify how much log data do you want to see. Options range from no logging to debug logging (this one will log absolutely everything it does; it's not useful unless you want to report bugs or debug the application).

Setting 3 - mild logging is recommended.</string>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>0 - no logging</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 - only critical errors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 - only errors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 - mild logging</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 - medium logging</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5 - excessive logging</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6 - log everything (debug output)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>When checked, Spdr will print all log messages to a log file, instead of using standard output/ standard error channels.</string>
        </property>
        <property name="text">
         <string>Use log file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="logPlaceholder" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Specify the file you wish to use as a log file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLog">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Specify the file you wish to use as a log file</string>
          </property>
          <property name="text">
           <string>Select dir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxSimulate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>When checked, Spdr will simulate it's usual workflow. What that means is that it will perform all the operations as usual, print all the log messages, etc. but it will not actually make any changes to the file system.

Simulation mode can be used to see if Spdr works correctly without risking any lasting changes to the data. It is best combined with using a high log level.</string>
        </property>
        <property name="text">
         <string>Simulate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelUpdateMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Update mode</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxUpdateMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Use this control to specify what should be done when Spdr encounters an already existing file with the same name present in output. If files are identical, Spdr will simply skip that file. But if the contents are different, &lt;b&gt;Update mode&lt;/b&gt; will kick in:
 - overwrite - will overwrite the file in output with the version from input
 - ignore - will skip the file
 - ask - will ask what to do (&lt;b&gt;NOT IMPLEMENTED YET!&lt;/b&gt;)</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Overwrite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ignore</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>170</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxLog</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonLog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
